app1.js

```diff
@@ -35,17 +35,17 @@ function dealInAbsolutes() {
     document.write(`<img src="{url}" alt="{alt}" />`)
 }
 
-function lightsaberColor(){
-    userAnswer = prompt('Blue or Red?');
+function lightSaberColor(){
+    let userAnswer = prompt('Blue or Red?');
 
-if(userAnswer.toLowerCase() === 'blue') {
-    let url = "https://mcdn.wallpapersafari.com/medium/10/39/TwfsOy.jpg"
-    document.write('<img src="' + url + '">')
-}
-else {
-    let url = "https://tvovermind.com/wp-content/uploads/2018/09/like-blood-from-a-stone-750x422.jpg"
-    document.write('<img src="' + url + '">')
-}
+    if(userAnswer.toLowerCase() === 'blue') {
+        let url = "https://mcdn.wallpapersafari.com/medium/10/39/TwfsOy.jpg"
+        document.write('<img src="' + url + '">')
+    }
+    else {
+        let url = "https://tvovermind.com/wp-content/uploads/2018/09/like-blood-from-a-stone-750x422.jpg"
+        document.write('<img src="' + url + '">')
+    }
```

Comment:
We talked about this one:
- Indent blocks to line up with function correctly.
- Add semi-colons at the end of your lines;
- Pull out `document.write` to be the last statement in the function.
- Usability and ADA dictates require an `alt` attribute on your image tag that gives a description of the image.